### ISSUE TITLE
time to remove `nlls` from PeakStats

### ISSUE THREAD

**Issue by prjemian** [bluesky/bluesky#1263]
As stated previously, the `nlls` attribute in PeakStats:
https://github.com/bluesky/bluesky/blob/f228b489a2fca3d22a5373e84f1ad3df5883860e/bluesky/callbacks/fitting.py#L218

is for a planned, undocumented, and unimplemented feature that has received no attention in a long time.  Its value is always a constant: `None`.  Its presence invites questions from new users as to why it is defined.

I propose this `nlls` attribute be removed since it is not useful.

**Comment by danielballan** [comment]
I support that. Got time to open a PR to strip it out?

**Comment by prjemian** [comment]
Sure.  Incurred a recent delay when VSCode editor [repeatedly crashes my VM](https://github.com/microsoft/vscode/issues/84745#issuecomment-553688580).  Introduces a certain discipline to save your work at every step and **not** in `/tmp`!

tl;dr: workaround in place and can pick this one up

**Comment by prjemian** [comment]
notes:

```
(base) mintadmin@mint-vm:~/.../eclipse/bluesky$ git grep -i nlls
bluesky/callbacks/best_effort.py:    ATTRS = ('com', 'cen', 'max', 'min', 'fwhm', 'nlls')
bluesky/callbacks/fitting.py:        self.nlls = None
bluesky/callbacks/fitting.py:        self.nlls = None
```

This is easy to complete.  Too easy.
